### ISSUE TITLE
Development progress?

### ISSUE THREAD

**Issue by pepperfighter** [clintbellanger/heroine-dusk#19]
I know that this repo hasn't been updated in a long time, and there's no information I could find about the development of the full version that was mentioned in the demo. I'm honestly a fan of this game and it's characteristics, so I'd just like to know what happened to development and whether or not it will be continued.

**Comment by minsc78** [comment]
It looks like Clint is not working on this anymore, but you could continue the story yourself: it does not look hard to do with the existing framework.

**Comment by Blueberryy** [comment]
Yeah just give Clint a rest, he's already made many for open-source
